docs/components/MessagingBubbleView.jsx

```diff
@@ -393,7 +403,7 @@ export default class MessagingBubbleView extends React.PureComponent {
               name: "replyTo",
               type: "React.ReactNode",
               description: "Optional reply-to announcement, if this DM was an announcement reply.",
-              option: true,
+              optional: true,
```

Comment:
Typo from before.

Reply:
wow, good catch, sorry I missed it in my review!